packages/curve-ui-kit/src/shared/ui/TokenIcons.tsx

```diff
@@ -20,10 +20,10 @@ export function TokenIcons({ tokens, ...props }: Props) {
       display="inline-grid"
       sx={{
         gridTemplateColumns: `repeat(${iconsPerRow}, auto)`,
-        [`& > *:nth-child(${iconsPerRow}n-1)`]: {
+        [`& > *:nth-of-type(${iconsPerRow}n-1)`]: {
```

Comment:
`*:nth-of-type` is basically the same as `*:nth-child` right? Wondering why you changed this